index.js

```diff
@@ -19,14 +19,19 @@ const baseLink = "https://www.icloud.com/shortcuts/";
  * A shortcut metadata.
  */
 class ShortcutMetadata {
-	constructor(object) {
-		// Not sure if we keep the native Apple names for the properties?
-		this.WFWorkflowClientRelease = object.WFWorkflowClientRelease;
-		this.WFWorkflowMinimumClientVersion = object.WFWorkflowMinimumClientVersion;
-		this.WFWorkflowIcon = object.WFWorkflowIcon;
-		this.WFWorkflowImportQuestions = object.WFWorkflowImportQuestions;
-		this.WFWorkflowTypes = object.WFWorkflowTypes;
-		this.WFWorkflowInputContentItemClasses = object.WFWorkflowInputContentItemClasses;
+	constructor(metadata) {
```

Comment:
I don't see actions in here. Can you add those, with a structure like below?

```json
{
    "actions": [
        {
            "identifier": "is.workflow.actions.date",
            "parameters": {
                "UUID": "FB44681C-9A30-43AC-A2F4-0B92272B5222"
            }
        }
    ]
}
```

Maybe make a `Action` class, and `<ShortcutMetadata>.actions` would be an array of them, with `identifier` being `WFWorkflowActionIdentifier` and `parameters` being `WFWorkflowActionParameters`. I don't think we should rename parameters, for an obvious reason (third-party apps).